### DISCUSSION TITLE
How do I avoid re-addition of deleted transaction when running import?

### DISCUSSION THREAD

**Discussion by cruzzoe** [firefly-iii/firefly-iii#9434] [Firefly III]
I have a transaction i want to delete. This is a Deposit. It deletes as expected. 

However, when I next run my import scripts, it adds back in the same transaction I just deleted. 

I'd expected the duplicate checking functionality to see that the hash for this existed previously. Note, I haven't clicked 'Purge all deleted records'. 

My recollection of how its supposed to work is that FF3 should see that this transaction was previously deleted, so avoid re-importing it. 

Is there something I can do to stop it re-adding in this transaction?

Note, I am using the ff3 data importer.

**Comment by JC5** [comment]
It should not add the transaction again if you just deleted it. You remember this correctly.

Do you also purge the data from your /profile page, perhaps?

**Comment by cruzzoe** [comment]
No purging was done. Not sure why it keeps coming back...Perhaps I've set
something odd for this in the account config. If my memory serves me
correctly, my other accounts dont have this behaviour. Something like
ignore_duplicates=False...

**Comment by cruzzoe** [comment]
Ok so I took a look at the import config that gets used by the docker image of the ff3_data_importer and I have:
  4     "duplicate_detection_method": "cell",
  3     "ignore_duplicate_lines": true,
  2     "unique_column_index": 17,
  1     "unique_column_type": "internal_reference",


In my other accounts, in the same importer file I have:
    "duplicate_detection_method": "cell",
  1     "ignore_duplicate_lines": true,
  2     "unique_column_index": 17,
  3     "unique_column_type": "internal_reference",

Some time ago we discussed the value:  "ignore_duplicate_transactions": false

 At the time, IIRC you said this wasn't used. Is this still the case/did I misunderstand?

**Reply by JC5** [reply]
That value has always been used. I could have explained it wrong, I apologize.
